kimchi/src/verifier_index.rs

```diff
@@ -19,13 +21,11 @@ use commitment_dlog::{
     srs::SRS,
 };
 use once_cell::sync::OnceCell;
-use oracle::poseidon::ArithmeticSpongeParams;
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
 use serde_with::serde_as;
-use std::io::SeekFrom::Start;
 use std::{
     fs::{File, OpenOptions},
-    io::{BufReader, BufWriter, Seek},
+    io::{BufReader, BufWriter, Seek, SeekFrom::Start},
```

Comment:
?

Reply:
it is used in line 276, I just merged the imports.